<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QuickModInstallDialog</class>
 <widget class="QDialog" name="QuickModInstallDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>800</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Installing mod...</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="dependencies">
      <attribute name="title">
       <string>Dependencies</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QListWidget" name="dependencyListWidget"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="downloadTab">
      <attribute name="title">
       <string>Downloads</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QSplitter" name="downloadSplitter">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="handleWidth">
          <number>6</number>
         </property>
         <widget class="QWidget" name="verticalLayoutWidget_2">
          <layout class="QVBoxLayout" name="downloadListLayout">
           <item>
            <widget class="QTreeWidget" name="progressList">
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
             <property name="alternatingRowColors">
              <bool>true</bool>
             </property>
             <property name="selectionMode">
              <enum>QAbstractItemView::NoSelection</enum>
             </property>
             <property name="rootIsDecorated">
              <bool>false</bool>
             </property>
             <property name="columnCount">
              <number>5</number>
             </property>
             <attribute name="headerDefaultSectionSize">
              <number>100</number>
             </attribute>
             <attribute name="headerShowSortIndicator" stdset="0">
              <bool>true</bool>
             </attribute>
             <attribute name="headerStretchLastSection">
              <bool>true</bool>
             </attribute>
             <column>
              <property name="text">
               <string>Name</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Version</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>URL</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Donations</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Progress</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="verticalLayoutWidget">
          <layout class="QVBoxLayout" name="webLayout">
           <item>
            <widget class="QTabWidget" name="webTabView">
             <property name="movable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QProgressBar" name="webModsProgressBar">
             <property name="maximum">
              <number>42</number>
             </property>
             <property name="value">
              <number>13</number>
             </property>
             <property name="format">
              <string>%v of %m web pages done</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="finishButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Finish</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="donateFinishButton">
       <property name="text">
        <string>Donate &amp;&amp; Finish</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>finishButton</sender>
   <signal>clicked()</signal>
   <receiver>QuickModInstallDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>566</x>
     <y>454</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>448</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
